### PR TITLE
Improved documentation

### LINKED ISSUE #6: The description is unclear
The description of the project is very unclear to me. What does "Verifiers looks like validators, but they are not bound to validation" mean?
I propose we change it to something more understandable.

### PR DESCRIPTION
**Pull request by JelF**
- Metioned rubygems.info
- Updated documentation in gemspec
- Improved gemspec code style (a bit)
- Updated pry dependency in gemspec to fix warnings
- Reworded description of gem (this should fix #6)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 [![Build Status](https://travis-ci.org/umbrellio/verifly.svg?branch=master)](https://travis-ci.org/umbrellio/verifly)
 
-This gem consists of several dependent components, which all could be
-used standalone. The most important one is [Verifier](#Verifier),
-but understanding [Applicator](#Applicator) and
-[ApplicatorWithOptions](#ApplicatorWithOptions) helps understand its API.
-The least important one, [ClassBuilder](#ClassBuilder), is only used in private APIs,
-but its own API is public
+This gem provides an api to run sequential checks like
+'ActiveModel::Validations'do, but with generic messages instead of errors.
```
Comment (akxcv) [inline README.md]: You forgot a space after `'ActiveModel::Validations'`.

File: README.md [hunk 2]
```diff
@@ -83,8 +84,8 @@ following:
 5. Change `.build_class` of other classes like `self if ...`.
 Don't change default implementation's `.build_class`
 
-6. Setup `.buildable_classes` on the abstract class, mentioning only direct chldren
-if you done step 4
+6. Setup `.buildable_classes` on the abstract class, mentioning only direct
+chldren if you done step 4
```
Comment (akxcv) [inline README.md]: ch**i**ldren

File: verifly.gemspec [hunk 3]
```diff
@@ -6,20 +6,29 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
...
-  spec.description   = ''
-  spec.homepage      = 'http://example.com'
+  spec.summary = <<~SUMMARY.gsub(/\s+/, '')
+    An api to run sequential checks like 'ActiveModel::Validations'
+    do, but with generic messages instead of errors
+  SUMMARY
 
-  spec.files         = Dir['lib/**/*.rb']
-  spec.executables   = []
+  spec.description = <<~DESCRIPTION.gsub(/\s+/, '')
```
Comment (akxcv) [inline verifly.gemspec]: `squish`?
Also, you'll have no spaces in this string.

File: verifly.gemspec [hunk 4]
```diff
@@ -6,20 +6,29 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
...
+  spec.homepage = 'https://github.com/umbrellio/verifly'
+  spec.license = 'MIT'
+
+  spec.files = Dir['lib/**/*.rb']
+  spec.executables = []
   spec.require_paths = %w[lib]
 
-  spec.add_development_dependency 'pry'
+  spec.required_ruby_version = '~> 2.3'
+  spec.add_development_dependency 'pry', '~> 0.10'
```
Comment (akxcv) [inline verifly.gemspec:31]: Why is `pry` a development dependency?

**Review by akxcv — CHANGES_REQUESTED** [review]

File: verifly.gemspec (hunk 3 above)
Comment (JelF) [reply verifly.gemspec]: `String#squish` is from ActiveSupport gem

File: verifly.gemspec (hunk 4 above)
Comment (JelF) [reply verifly.gemspec:31]: Because it used in development? It required in bin/console and spec/spec_helper.rb

File: verifly.gemspec (hunk 3 above)
Comment (JelF) [reply verifly.gemspec]: Ill fix it to `gsub(/\s+/, ' ')`, it was a bug

File: verifly.gemspec (hunk 4 above)
Comment (JelF) [reply verifly.gemspec:31]: We also have a static check for absence of `binding.pry` before specs would start

**Review by akxcv — APPROVED** [review]

**Pull request merged**